src/io/leopard/toLeopard.ts

```diff
@@ -4,7 +4,9 @@ import Block from "../../Block";
 import * as BlockInput from "../../BlockInput";
 import { OpCode } from "../../OpCode";
 
-import * as prettier from "prettier";
+// TODO: Upgrade to Prettier v3, which formats asynchronously
+// Or https://github.com/prettier/prettier-synchronized if it's OK to be Node-only
+import * as prettier from "prettier2";
```

Comment:
prettier-synchronized sounds a little evil (with due respect!). It makes sense not to bother with prettier 2->3 (at runtime) in this PR, but is this something we could follow up on pretty quickly? 

The only frontends that I believe actually interface with `toLeopard` are sb-edit's newish CLI (#129) and leopard-website's [`exportProjectWithBufferAssetsToZip`](https://github.com/PullJosh/leopard-website/blob/b6d56ef7d8adc35f83c603eaf6ef77a9b62fa1c5/lib/convertProject.ts#L37), which is trivial to async-ify (it is already evaluated only in async contexts).

Reply:
I can create a PR to leopard-website, and I'll update to prettier v3 here after that goes in